src/app/shared/top-panel/user-panel/notifications-menu/notifications-menu.component.html

```diff
@@ -0,0 +1,200 @@
+<div class="panel-item mx-2 notifications">
+  <div class="dropdown show">
+    <a href="#"
+       role="button"
+       id="notificationsDropdownLink"
+       data-toggle="dropdown"
+       data-offset="0,10"
+       aria-haspopup="true"
+       aria-expanded="false">
+
+      <span class="fa-stack">
+        <i class="fa fa-bell fa-stack-2x"
+           [class.text-primary]="(unreadNotifications$ | async)?.length > 0"
+           [class.text-secondary]="!(unreadNotifications$ | async) || (unreadNotifications$ | async).length === 0"
+        ></i>
+        <span class="fa-stack-1x" *ngIf="(unreadNotifications$ | async)?.length > 0">
```

Comment:
If you subscribe to the same observable many times in a template, you can use a little trick. Subscribe to it like this in their common ancestor like this:
```html
<div *ngIf="{unread: unreadNotifications$ | async, all: notifications$ | async} as notifications">
  ...
```
... and then use reference them as `notifications.unread` and `notifications.all` where you need them.